static_analyzer/engine/adapters/typescript_adapter.py

```
@@ -10,6 +10,8 @@
 
 class TypeScriptAdapter(LanguageAdapter):
 
+    _JSX_LANGUAGE_IDS = {".tsx": "typescriptreact", ".jsx": "javascriptreact"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)